_ClientSideUI/index.html

```diff
@@ -336,14 +336,14 @@
     var metricsDef;        //holds the definition of the metrics (from server side metrics.js)
     var eventsDef;
     var showHiddenNodes=false, showRawSend=false;
-    var socket = io.connect('<?php echo $_SERVER['HTTP_HOST']; ?>', {'connect timeout': 1000}); //limit chrome xhr-polling fall back delay
```

Comment:
Actually it works fine with current versions of socket.io. I initially changed it to use pure javascript `location.host` like I had done in other projects of mine but then I read the 1.x docs and found that wasn't even needed any more. My current home project is running with this change. One difference may be that in reworking the patches on my local branch I left one possibly key change out. The client side javascript likely needs to be loaded locally and not from the CDN. My current home branch that contains a few other patches is running from the local socket.io libraries and not the CDN.